src/renderer/features/wallets/ForgetWallet/model/forget-wallet-model.ts

```diff
@@ -4,6 +4,7 @@ import { AccountId, MultisigAccount, Wallet } from '@shared/core';
 import { accountUtils, walletModel, walletUtils } from '@entities/wallet';
 import { useBalanceService } from '@entities/balance';
 import { useForgetMultisig } from '@entities/multisig';
+import { wcDetailsModel } from "@widgets/WalletDetails/model/wc-details-model";
```

Comment:
⚠️ Widget inside feature is not allowed ⚠️
Handle balance deletion inside `wc-details-model`

Reply:
fixed